dbt_subprojects/solana/models/tokens/solana/tokens_solana_transfers.sql

```diff
@@ -76,7 +126,55 @@ SELECT
     , outer_executing_account
 FROM base tr
 --get token and accounts
-LEFT JOIN {{ ref('solana_utils_token_accounts') }} tk_s ON tk_s.address = tr.from_token_account
+LEFT JOIN {{ ref('solana_utils_token_accounts') }} tk_s ON tk_s.address = tr.from_token_account 
 LEFT JOIN {{ ref('solana_utils_token_accounts') }} tk_d ON tk_d.address = tr.to_token_account
 WHERE 1=1
--- AND call_block_time > now() - interval '90' day --for faster CI testing
\ No newline at end of file
+{% if is_incremental() %}
+AND {{incremental_predicate('block_time')}}
+{% endif %}
+{% if not is_incremental() %}
+AND block_time > now() - interval '7' day
+{% endif %}
```

Comment:
Do we need to remove those `if not is_incremental()` clauses?

Reply:
ye definitely, just still testing this is to make sure the trial runs don't time out